### ISSUE TITLE
CLI module missing Jetbrains @Nullable annotation dependency needed in runtime

### ISSUE THREAD

**Issue by mplescano** [bbottema/simple-java-mail#270]
command:  ./bin/sjm --help
version: cli-module-6.0.4-standalone-cli.zip
java version: openjdk version "1.8.0_252"
OpenJDK Runtime Environment (build 1.8.0_252-8u252-b09-1~18.04-b09)
OpenJDK 64-Bit Server VM (build 25.252-b09, mixed mode)

```
root@mosip-sandbox-v1:/home/mplescano/bin# cd ..
root@mosip-sandbox-v1:/home/mplescano# ./bin/sjm --help
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
Exception in thread "main" java.lang.NoClassDefFoundError: org/jetbrains/annotations/Nullable
        at org.simplejavamail.internal.clisupport.BuilderApiToPicocliCommandsMapper.getArgumentsForCliOption(BuilderApiToPicocliCommandsMapper.java:237)
        at org.simplejavamail.internal.clisupport.BuilderApiToPicocliCommandsMapper.generateOptionsFromBuilderApi(BuilderApiToPicocliCommandsMapper.java:170)
        at org.simplejavamail.internal.clisupport.BuilderApiToPicocliCommandsMapper.generateOptionsFromBuilderApiChain(BuilderApiToPicocliCommandsMapper.java:135)
        at org.simplejavamail.internal.clisupport.BuilderApiToPicocliCommandsMapper.generateOptionsFromBuilderApi(BuilderApiToPicocliCommandsMapper.java:175)
        at org.simplejavamail.internal.clisupport.BuilderApiToPicocliCommandsMapper.generateOptionsFromBuilderApiChain(BuilderApiToPicocliCommandsMapper.java:135)
        at org.simplejavamail.internal.clisupport.BuilderApiToPicocliCommandsMapper.generateOptionsFromBuilderApi(BuilderApiToPicocliCommandsMapper.java:123)
        at org.simplejavamail.internal.clisupport.CliSupport.<clinit>(CliSupport.java:37)
        at org.simplejavamail.cli.SimpleJavaMail.main(SimpleJavaMail.java:31)
Caused by: java.lang.ClassNotFoundException: org.jetbrains.annotations.Nullable
        at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
        ... 8 more
```

**Comment by bbottema** [comment]
I found that the dependency was misconfigured for Jetbrains nullability annotations. Normally it would be correct like this, but Simple Java Mail actually uses the classes in runtime using reflection.

I'll release a fix in some time, but in the meantime you can easily fix this by dropping the appropriate jar in the lib folder.

- https://repo1.maven.org/maven2/com/github/bbottema/jetbrains-runtime-annotations/1.0.0/jetbrains-runtime-annotations-1.0.0.jar

**Comment by bbottema** [comment]
I just released 6.0.5 which should fix it.
